labs/iot/gps-lab/steps/set-up-azure-maps.md

```diff
@@ -0,0 +1,23 @@
+# Set Up Azure Map
+
+In this step you will set up Azure Map.
+
+## Steps
+
+1. Go to [Azure portal]('https://portal.azure.com') and login to your subscription. 
```

Comment:
Can this be replaced with docs links to re-use existing docs content?

Reply:
I think, as with many cases - How to set up pi for example - or how to delete a resource/ resource group - we are writing the steps ourselves we should also write this step ourselves to be consistent.  However I added a more info to point them to the official documentation.